FastMember/MemberSet.cs

```diff
@@ -11,11 +11,9 @@ namespace FastMember
     public sealed class MemberSet : IEnumerable<Member>, IList<Member>
     {
         Member[] members;
-        internal MemberSet(Type type)
+        internal MemberSet(MemberInfo[] members)

```

Comment:
This set of members is filtered, so we want to avoid repeating the same reflection logic.